lib/jekyll-plugin-platoniq-journal/numbers_tag.rb

```diff
@@ -47,21 +47,21 @@ def include_file_path
     def render_all
       output = []
 
-      output << %(<ul class="pj-numbers">)
+      output << %(<dl class="pj-numbers">)
       jdata["items"].each do |item|
         output << <<~NUM
-          <li>
-            <strong>
-              #{item["number"]}
-            </strong>
-            <span class="pj-number--description">
-              <span class="pj-number--icon">#{@icon}</span>
+          <div class="pj-numbers--item">
```

Comment:
.. and this a `<dl>`?
```suggestion
          <dl class="pj-numbers--item">
```

Reply:
I thought about that, but I'm not sure what's best... So if you think it's better to have many `dl` elements nested inside a `div`, I can change it.

Reply:
Yes, could you change it please, in my opinion it makes more sense 😄